### PR TITLE
	modified:   Dockerfile

### PR DESCRIPTION
**Pull request by jwillmartin**
modified:   src/v2i-hub/MessageLoggerPlugin/manifest.json
	modified:   src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.cpp
	modified:   src/v2i-hub/MessageLoggerPlugin/src/MessageLoggerPlugin.h
	deleted:    src/v2i-hub/ODEPlugin/CMakeLists.txt
	deleted:    src/v2i-hub/ODEPlugin/manifest.json
	deleted:    src/v2i-hub/ODEPlugin/src/ODEPlugin.cpp
	deleted:    src/v2i-hub/ODEPlugin/src/ODEPlugin.h
	deleted:    src/v2i-hub/SPaTLoggerPlugin/CMakeLists.txt
	deleted:    src/v2i-hub/SPaTLoggerPlugin/manifest.json
	deleted:    src/v2i-hub/SPaTLoggerPlugin/src/SPaTLoggerPlugin.cpp
	deleted:    src/v2i-hub/SPaTLoggerPlugin/src/SPaTLoggerPlugin.h

# PR Details
## Description
ODELoggerPlugin removed due to one-off use-case and another plugin (ODEForwardPLugin) with updated functionality.
SPATLoggerPlugin removed and functionality merged into MessageLoggerPlugin. This reduces number of plugins and corrects description in MessageLogger.

## Related Issue
https://github.com/usdot-fhwa-OPS/V2X-Hub/issues/425

## Motivation and Context

These changes remove the unused ODEPlugin and merges the functionality of the SPaTLogger into the MessageLogger.
The MessageLogger plugin now logs both received SPaT and BSM messages in the same log file.

## How Has This Been Tested?

Individually sent messages to the SPAT and MessageReceiver plugins to have SPaT and BSMs logged. Then send both message types at the same time. MessageLogger plugin was able to log both individually and simultaneously into the same log file. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,8 @@ The V2X Hub system reduces time needed to create and deploy a roadside based V2X
 * ODE Forward Plugin – Pushes data to an operational data environment server using a known IP address and port.
 * Pedestrian Plugin – Creates a PSM using information obtained from nomadic devices (ex. cell phones) through a local webserver.
 * Preemption Plugin – Calls a preemption table on a controller using NTCIP 1202 V3 commands to provide passage to an emergency vehicle upon request through BSMs
-* SPAT Logger Plugin – Logs V2X Hub generated, UPER-encoded SPAT messages in a .csv file along with a system timestamp
 * TIM Plugin – Creates and broadcasts a TIM message from an .xml file based on user input through GUI or local webserver
-* BSM Logger Plugin – Decodes and logs BSMs received by the Message Receiver Plugin to a .csv file
+* Message Logger Plugin – Decodes and logs BSMs and SPaTs received by the Message Receiver Plugin to a .json file
```
Comment (paulbourelly999) [inline README.md:32]: Spat messages are not received by the message receiver plugin. Maybe remove that part of the sentence. Also doesn't it create a json and a hex encoded file.?

**Review by paulbourelly999 — COMMENTED** [review]
Looks good other than the mentioned comment

File: README.md (hunk 1 above)
Comment (jwillmartin) [reply README.md:32]: The hex is in the json, but there's also a bin file that's generated. Made the changes and pushed

**Review by paulbourelly999 — APPROVED** [review]
Looks good

**Pull request merged**